mypy/nodes.py

```diff
@@ -103,7 +103,8 @@ def get_column(self) -> int:
     '__doc__': None,  # depends on Python version, see semanal.py
     '__path__': None,  # depends on if the module is a package
     '__file__': '__builtins__.str',
-    '__package__': '__builtins__.str'
+    '__package__': '__builtins__.str',
+    "__annotations__": "__builtins__.dict",
```

Comment:
Note that all other keys and values use `'` instead of `"`

Reply:
Woopsie, will fix! 👍